langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -56,6 +68,10 @@ public isolated function iterator(map<Type> m) returns object {
 
 # Returns the member of a map with given key.
 #
+# ```
+# {"Carl": 85, "Bob": 50, "Max": 60}.get("Carl") ⇒ 85
+# ```
+#
 # This for use in a case where it is known that the map has a specific key,
```

Comment:
Examples have to be just before the parameter descriptions, not in between the function description. Please fix everywhere.